int8 count_markers
MarkerInImage[] markers
